docs/commands/test.md

```diff
@@ -142,7 +142,16 @@ before those paths are to be considered 'visible' to the module loader. If a
 given module's path matches any of the patterns, it will not be `require()`-able
 in the test environment.
 
-#### testPathIgnorePatterns
+### package
+
+Default: `undefined`
+
+Run all the test for the workspace with the specified package name. Can be
+repeated to select more than one workspace. Can be combined with the
```

Comment:
Also a typo?

```suggestion
repeated to select more than one package. Can be combined with the
```

Reply:
Maybe not. I'm confused - are they workspaces or packages? In my mind, a workspace can consist of many packages (the workspace usually means the `packages` dir, if talking in yarn languge)